.gitignore

```diff
@@ -49,13 +49,12 @@ Temporary Items
 # =========================
 # Build Outputs
 # =========================
-
 *.gba
 *.sav
 *.elf
 *.map
 *.bin
-!data/**/*.bin
+data/**/*.bin
```

Comment:
I think you can just remove this line as it's covered by `*.bin`

Reply:
Yup, did that now!